src/runtime-rs/crates/resource/src/manager_inner.rs

```diff
@@ -291,6 +292,40 @@ impl ResourceManagerInner {
                         devices.push(agent_device);
                     }
                 }
+                "c" => {
```

Comment:
I think the device type "c" can be a real char device, I think need to add more checks for the Vfio device.

Reply:
Yes, you're right, more check added to filter only for vfio device